versions.json

```
@@ -60,11 +60,15 @@ layout: null
       "instead": "2.8"
     },
     "2.7": {
-      "latest": "{{ page.previous_akka_version }}",
+      "latest": "2.7.0",
       "outdated": true,
-      "instead": "2.8"
+      "instead": "2.9"
     },
     "2.8": {
+      "latest": "2.8.5",
```

Shouldn't 2.8 be marked as outdated too?

```suggestion
      "latest": "2.8.5",
      "outdated": true,
```

---

No, as it still is supported.

---

When do we add the warning "is outdated and not supported!"? That wouldn't be correct for 2.8.
We have probably used that for 2.7 even though it has been supported. https://doc.akka.io/docs/akka/2.7/index.html

I was hoping `instead` would show the warning, but without the "outdated"?

---

For the record the warning logic has now been changed to match the Alpakka version boxes across Akka and Akka HTTP